content/ux-guide/design/build-a-prototype.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Build a prototype
+description: "18F encourages building something that will help you answer your questions with the least investment."
+permalink: /ux-guide/design/build-a-prototype/
+layout: layouts/page
+tags: ux-guide
+eleventyNavigation: 
+    parent: ux-guide-design
+    key: ux-guide-design-prototype
+    order: 2
+    title: Build a prototype
+sidenav: true
+sticky_sidenav: true
+subnav:
+- text: Communicating with prototypes
+  href: '#communicating-with-prototypes'
+- text: Building prototypes with Cloud.gov Pages
+  href: '#building-prototypes-with-cloud.gov-pages'
+- text: Getting started with Cloud.gov Pages
+  href: '#getting-started-with-cloud.gov-pages'
+- text: Authorized prototyping tools
+  href: '#authorized-prototyping-tools'
+---
+{% comment %}
+Need to update all the Methods links once that's replatformed too.
+{% endcomment %}
+
+Prototypes are useful to inform a new concept, identify how to refactor an existing product feature, service, or process, and help uncover current development unknowns.
+
+Use prototypes to:
+
+- Align the team (“What problem are we solving?”)
+- Further specify elements of the design (“Should it look like this, or that?”)
+- Demonstrate that your ideas are technically possible
+- Explore/set up the deployment process
+- Demonstrate a collaborative design process
+- Reduce risk (“What are the riskiest assumptions and how can we test them?”)
+- Validate a [design hypothesis [18F design methods]](https://methods.18f.gov/decide/design-hypothesis/)
```

Comment:
Maybe worthwhile to make methods a '/TODO/' link because we are going to replatform them as well.